documentation/experimental_features.md

```diff
@@ -6,7 +6,136 @@ while iterating on new functionality. Almost all experimental features are
 eventually stabilized in future releases. While a feature is experimental, its
 API might change, requiring the user to update their code or configuration. The
 Bolt team attempts to make these changes painless by providing useful warnings
-around breaking behavior where possible. 
+around breaking behavior where possible.
+
+## Configuration policies and the policy command
+
+This feature was introduced in [Bolt 3.21.0](). 
+
+### What are policies?
+
+Configuration policies, or policies for short, are public classes that can be
+applied directly to one or more targets. Policies are Puppet code stored in the
+`manifests/` directory of modules on the modulepath. They can be applied
+directly to targets just like other Puppet code.
+
+### Policies in a Bolt project's configuration file
+
+The project configuration file, `bolt-project.yaml`, supports a `policies` key
+which lists the policies available to a project. A Puppet class becomes a policy
+when it's listed under the `policies` key, and if the `policies` key is not
+configured then no policies will be available for Bolt to apply.
+
+Policies may be manually added to the `policies` key in `bolt-project.yaml` or
+managed via Bolt commands. This key supports glob pattern for easily listing
+multiple policies, i.e. boltproject::* includes boltproject::admin and
+boltproject::sshkeys policies.
```

Comment:
```suggestion
multiple policies. For example, `bolt::project::*` includes `bolt::project::admin` and
`bolt::project::sshkeys` policies.
```

We should also make sure the doc is consistent between `boltproject` and `bolt::project` prefixes.

Reply:
Missed this, thanks.